web/src/app/administration/workbasket/details/information/workbasket-information.component.ts

```diff
@@ -280,16 +280,16 @@ implements OnInit, OnChanges, OnDestroy {
         response => {
           this.requestInProgressService.setRequestInProgress(false);
           this.workbasketService.triggerWorkBasketSaved();
-          if (response) {
+          if (response.status === 202) {
             // new Key ERROR_TYPES.MARK_ERR
```

Comment:
Please mark this key with a `TODO: message changed` so that I'll find it during my work on TSK-931.

Reply:
This is still not done and important. :)